src/Draco.Compiler.Tasks/Draco.Compiler.Tasks.csproj

```diff
@@ -1,13 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>11</LangVersion>
     <IsPackable>false</IsPackable>
+    <Nullable>disable</Nullable>
```

Comment:
Because we set `<Nullable>enable</Nullable>` in Directory.Build.props now, it was disabled before. This is netstandard2.0 so nullable does not quite work